jupyter_server/services/contents/filemanager.py

```diff
@@ -26,80 +26,34 @@
 from traitlets import Any, Unicode, Bool, TraitError, observe, default, validate
 from ipython_genutils.py3compat import getcwd, string_types
 
-from notebook import _tz as tz
-from notebook.utils import (
+from jupyter_server import _tz as tz
+from jupyter_server.utils import (
     is_hidden, is_file_hidden,
     to_api_path,
 )
-from notebook.base.handlers import AuthenticatedFileHandler
-from notebook.transutils import _
+from jupyter_server.base.handlers import AuthenticatedFileHandler
+from jupyter_server.transutils import _
 
 try:
     from os.path import samefile
 except ImportError:
     # windows + py2
-    from notebook.utils import samefile_simple as samefile
+    from jupyter_server.utils import samefile_simple as samefile
 
 _script_exporter = None
 
 
-def _post_save_script(model, os_path, contents_manager, **kwargs):
-    """convert notebooks to Python script after save with nbconvert
-
-    replaces `jupyter notebook --script`
-    """
-    from nbconvert.exporters.script import ScriptExporter
-    warnings.warn("`_post_save_script` is deprecated and will be removed in Notebook 5.0", DeprecationWarning)
-
-    if model['type'] != 'notebook':
-        return
-
-    global _script_exporter
-    if _script_exporter is None:
-        _script_exporter = ScriptExporter(parent=contents_manager)
-    log = contents_manager.log
-
-    base, ext = os.path.splitext(os_path)
-    script, resources = _script_exporter.from_filename(os_path)
-    script_fname = base + resources.get('output_extension', '.txt')
-    log.info("Saving script /%s", to_api_path(script_fname, contents_manager.root_dir))
-    with io.open(script_fname, 'w', encoding='utf-8') as f:
-        f.write(script)
-
-
```

Comment:
Should this be kept?

Reply:
Can you provide more context on this one?  Do you mean the whole file?  The diff doesn't show up anymore.